packages/kbn-management/settings/components/field_row/__stories__/select_field.stories.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { getFieldRowStory, getStory } from './common';
+
+const argTypes = {
```

Comment:
Can you explain why we pass this to `getFieldRowStory()`? It seems to overwrite the `value` story arg with this object rather than keeping the setting's default value there:

<img width="1706" alt="Screenshot 2023-09-08 at 18 17 59" src="https://github.com/elastic/kibana/assets/59341489/d1ac626e-4774-4655-8922-8622a5cfc7c0">

Reply:
Nice catch!  Fixed.